src/templates/haml/home5/index.html.haml

```diff
@@ -0,0 +1,594 @@
+                %span#virtual-fields
+                  %span.acc-input{:style=>'margin:0px 0 0 0'}
+                    %label.grd-grid-4.grd-grid-phablet-12.grd-grid-tablet-12.grd-grid-mobile-12{:for => 'Email',:style=>'padding:0;margin-top:0px;text-align: center;padding-right: 0px;'}= l('Your Email:')
+                    %span.grd-grid-8.grd-no-gutter.grd-no-gutter.grd-grid-phablet-12.grd-grid-tablet-12.grd-grid-mobile-12{:style=>'height:40px;overflow:hidden;'}
+                      %input.import-fb-email-field.grd-grid-10{:name => 'Email', :id => 'Email', :type => 'email', :required => true, :placeholder=>'your email here':style=>'width:83.33333%;text-align: left;'}
+                      .import-fb-email-btn.grd-grid-2{:title=>'Import Your Email From Facebook',:style=>'cursor:pointer; background:#3b5998;height:100%;'}
+                        %img{:src=>url_for('images/pages/home5/facebook-letter.svg'),:style=>'max-height: 100%;max-width: 100%;margin-top: 3px;'}
+                  %span.acc-input{:style=>'margin-top:-9px'}
+                    %label.grd-grid-4.grd-grid-phablet-12.grd-grid-tablet-12.grd-grid-mobile-12{:for => 'chooseapassword',:style=>'padding:0;margin:0px 0 0 0;text-align: center;padding-right: 0px;'}= l('Password:')
+                    %span.grd-grid-8.grd-no-gutter.grd-no-gutter.grd-grid-phablet-12.grd-grid-tablet-12.grd-grid-mobile-12
+                      %input#chooseapassword{:name => 'chooseapassword', :type => 'password', :maxlength => '25', :required => true,:style=>''}
+                  %span.acc-input{:style=>'margin-top:9px'}
+                    .grd-grid-12{:style=>'margin:0px;padding:0px'}
+                      %button{:id => 'no_btn_registration',:style=>'margin:0px',:class=>'input-button button'}
+                        != l('Create free<br/>account now')
+                %input#actype{:type => 'hidden', :name => 'actype', :value => 'virtual'}
+                %input{:type => 'hidden', :name => 'l', :value => $lang}
+              %p{:style=>'margin:0px;padding:0px;line-height: 1.5em;font-weight: bold;'}= l('Get your free $10k virtual money account')
+        .bottom-dark-box.acc-input-security{:style=>'margin:0 0 0 -9px;overflow:hidden;bottom:0;padding: 0px 0;background:#15212d;font-size: 72%;width:100%;text-align:center;color:#fff'}
+          %p{style=>'color:inherit; padding: 1px 8px;line-height: 1.3em;font-size: 0.9em;'}
+            %img.acc-input-security-icon{:src=>url_for('images/pages/home5/security-w.svg'),:style=>'margin: -6px 5px 0;max-width:13px;'}
+            = l('we use highest security and privacy measures')
+  .seen-on.section-margin
+    .grd-container
+      .grd-grid-6.grd-hide-tablet.grd-hide-phablet.grd-hide-mobile{:style=>'padding-top:8px'}
+        %h4{:style=>'margin:53px 0 28px;line-height:1.7em;padding: 0 0px;'}= '1st in the market and trusted by over 800.000 users'
```

Comment:
Can we replace '.' with ',' so 800.000 will be 800,000 as we follow this convention